src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureColor {
```

Comment:
```suggestion
public class IsoscelesTrapezoid extends Figure {
```
**Same for other classes.**